src/attr/_make.py

```diff
@@ -1196,6 +1208,13 @@ def attrs(
         If a list of callables is passed, they're automatically wrapped in an
         `attr.setters.pipe`.
 
+    :param field_transformer:
+        A function that is called with the original class object and all
+        fields: ``ft(cls: type, attribs: List[Attribute]) -> List[Attribute]``.
+        The hook is called right before the class gets transformed into an
+        attrs class.  The fields are already :class:`Attribute` instances.  You
+        can use this, e.g., to automatically add converters or validators to
+        fields.
```

Comment:
I'm gonna say we should shorten this to "what does it do and why it's interesting" and then link to extending:

```suggestion
        A function that is called with the original class object and all
        fields right before ``attrs`` finalizes the class.  You can use
        this, e.g., to automatically add converters or validators to
        fields based on their types.  See `transform-fields` for more details.
```